packages/fxa-content-server/server/lib/amplitude.js

```diff
@@ -244,6 +244,66 @@ const EVENTS = {
     group: GROUPS.registration,
     event: 'domain_validation_ignored',
   },
+
+  'screen.get-started': {
```

Comment:
I spent way too much time debugging why I couldn't make these "fuzzy" events. In the end I punted and just manually defined the metrics for each page 🤷🏽‍♂️